ui/admin/app/components/form/host-set/add-hosts/index.js

```diff
@@ -41,14 +41,14 @@ export default class FormHostSetAddHostsIndexComponent extends Component {
 
   /**
    * Toggle host selection
-   * @param {HostModel} host
+   * @param {string} hostId
    */
   @action
-  toggleHost(host) {
-    if (!this.selectedHostIDs.includes(host.id)) {
-      this.selectedHostIDs.addObject(host.id);
+  toggleHost(hostId) {
+    if (!this.selectedHostIDs.includes(hostId)) {
+      this.selectedHostIDs.addObject(hostId);
     } else {
-      this.selectedHostIDs.removeObject(host.id);
+      this.selectedHostIDs.removeObject(hostId);
```

Comment:
I refactored this action (and other similar ones) to just use the resource id as the parameter instead of the model. Since only the id is really necessary. Thoughts ?